### PR TITLE
Storage Overlay use /backpack

### PR DESCRIPTION
**Pull request by IRONM00N**
idk if I did this correctly

### CODE REVIEW

**Review by romangraef — COMMENTED** [review]
Does this break the normal clicking on chests in the gui? will test tomorrow

**Comment by CalMWolfs** [comment]
works fine for me, you can also remove the following file:
src/main/kotlin/io/github/moulberry/notenoughupdates/commands/help/StorageViewerWhyCommand.kt

**Review by NopoTheGamer — DISMISSED** [review]

File: src/main/kotlin/io/github/moulberry/notenoughupdates/commands/dev/SimpleDevCommands.kt [hunk 1]
```diff
@@ -66,7 +64,7 @@ class SimpleDevCommands {
                 }
             }.withHelp("Disable the neu storage overlay")
             thenExecute {
-                NotEnoughUpdates.INSTANCE.config.storageGUI.enableStorageGUI3 = true
+                NotEnoughUpdates.INSTANCE.config.storageGUI.enableStorageGUI3 = false
```
Comment (romangraef) [inline src/main/kotlin/io/github/moulberry/notenoughupdates/commands/dev/SimpleDevCommands.kt:67]: This disables the storage when typing `/neuenableStorage`

**Review by romangraef — CHANGES_REQUESTED** [review]

File: src/main/kotlin/io/github/moulberry/notenoughupdates/commands/dev/SimpleDevCommands.kt [hunk 2]
```diff
@@ -66,7 +64,7 @@ class SimpleDevCommands {
                 }
             }.withHelp("Disable the neu storage overlay")
```
Comment (IRONM00N) [inline src/main/kotlin/io/github/moulberry/notenoughupdates/commands/dev/SimpleDevCommands.kt:65]: Should this not be "Enable the neu storage overlay"?

**Comment by jani270** [comment]
You might wanna add a very small cooldown between switching pages/after switching a few pages so this doesnt happen: "A kick occurred in your connection, so you were put in the SkyBlock lobby!"

**Comment by CalMWolfs** [comment]
> You might wanna add a very small cooldown between switching pages/after switching a few pages so this doesnt happen: "A kick occurred in your connection, so you were put in the SkyBlock lobby!"

you can add these modified lines back in for this where they were before
public long storageOpenSwitchMillis = 0;
if (System.currentTimeMillis() - storageOpenSwitchMillis < 100) return;

**Review by romangraef — APPROVED** [review]

**Review by NopoTheGamer — APPROVED** [review]

**Pull request merged**